source/draw/gpu/opengl/OpenGLContext.ooc

```diff
@@ -38,13 +38,32 @@ OpenGLContext: class extends GpuContext {
 	_pointsShader: OpenGLMap
 	_meshShader: OpenGLMapMesh
 	_renderer: GLRenderer
-	_recycleBin := _RecycleBin new()
+	_recycleBinMonochrome: RecycleBin<OpenGLMonochrome>
```

Comment:
The downside of that is that it's harder to limit to exactly 15 of each type. We would recycle 60 of any type.